sdk/src/client/bindings/python/examples/02_generate_addresses.py

```diff
@@ -8,49 +8,40 @@
 secret_manager = MnemonicSecretManager("endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river")
 
 # Generate public address with default account index and range.
-addresses = client.generate_addresses(secret_manager, options={})
+addresses = client.generate_addresses(secret_manager)
 
 print('List of generated public addresses:', *addresses, sep='\n')
 print()
 
 # Generate public address with custom account index and range.
-addresses = client.generate_addresses(secret_manager, {
-    "accountIndex": 0,
-    "range": {
-        "start": 0,
-        "end": 4,
-    },
-})
+addresses = client.generate_addresses(secret_manager, 
+    account_index=0, 
+    start=0, 
+    end=4)
 
 print('List of generated public addresses:', *addresses, sep='\n')
 print()
 
 # Generate internal addresses with custom account index and range.
-addresses = client.generate_addresses(secret_manager, {
-    "accountIndex": 0,
-    "range": {
-        "start": 0,
-        "end": 4,
-    },
-    "internal": True,
-})
+addresses = client.generate_addresses(secret_manager, 
+    account_index=0, 
+    start=0, 
+    end=4, 
+    internal=True)
 
 print('List of generated internal addresses:', *addresses, sep='\n')
 print()
 
 # Generate addresses with providing all inputs, that way it can also be done offline without a node.
-addresses = client.generate_addresses(secret_manager, {
-    "coinType": 4219, # Shimmer coin type
-    "accountIndex": 0,
-    "range": {
-        "start": 0,
-        "end": 4,
-    },
-    "internal": False,
+addresses = client.generate_addresses(secret_manager, 
+    coin_type=4219, # Shimmer coin type
```

Comment:
Should we add constants for IOTA and Shimmer coin type to the Python binding?

Reply:
Yeah, I thought about it, but seems like I forgot it 😅 .
Will update this PR